client/state/notices/README.md

```diff
@@ -0,0 +1,53 @@
+Notices
+===========
+
+A module for managing global notices.
+
+## Create a notice
+
+Provides some helper functions to render a notice on the UI. Types of notices are `error`, `success`.
+
+### How to use?
+
+#### Connect to Redux
+First, the component that wants to send notices must be connected to Redux.
+
+```javascript
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import { successNotice, errorNotice } from 'state/notices/actions';
+
+...
+
+export default connect(
+	null,
+	dispatch => bindActionCreators( { successNotice, errorNotice }, dispatch )
+)( Component );
+```
+
+#### Uses
+
+Now you can use notices like so:
+
+```javascript
+
+this.props.successNotice(
+    'Settings saved successfully!', {
+    	duration: 4000
+} );
+
+```
+
+
+#### The available methods are:
+
+* `notices.success()`
+* `notices.error()`
```

Comment:
Thunks:
So they are micromanaging action creators. They are declared as action creators that come in and say
> I know what I'm doing here, please dont try to help me, I have this handled.

They are used primarily in asynchronous actions, because action creator immidiately returns plain object, theres no place for promises, callbacks, etc. And even if there were, you need to have `dispatch` available.

So, thunks have access to both `dispatch` and `state` and can do asynchronous stuff.

I needed to use a thunk in `createNotice` because it can set a timeout for a notice do disappear and to perform this action after a timeout, needs access to `dispatch`.

`removeNotice` does not have any asynchronous actions, so it can be a plain action creator. It just kills a notice with a certain id